main/tasks/asic_task.c

```diff
@@ -7,28 +7,16 @@
 
 static const char *TAG = "ASIC_task";
 
-static bm_job ** active_jobs;
-
 void ASIC_task(void * pvParameters)
 {
 
     GlobalState *GLOBAL_STATE = (GlobalState*)pvParameters;
 
-
     uint8_t buf[CHUNK_SIZE];
     memset(buf, 0, 1024);
 
     uint8_t id = 0;
 
-    active_jobs = malloc(sizeof(bm_job *) * 128);
```

Comment:
active jobs is used to maintain all of the recent jobs sent to the asic and the nonce packets returned can then be mapped back to this list.